### PR TITLE
Feature/len actions 2

### PR DESCRIPTION
**Pull request by antazoey**
New actions:

* Add high risk employee
* Remove high risk employee
* List departing employees
* List high risk employees

I have tested both with the Action menu as well as in Playbooks

### CODE REVIEW

File: code42.json [hunk 1]
```diff
@@ -61,11 +61,129 @@
             "output": [],
             "versions": "EQ(*)"
         },
+        {
+            "action": "add departing employee",
```
Comment (antazoey) [inline code42.json:65]: Diff looks a little funny because I moved the `add` commands above the `remove`

File: code42.json [hunk 2]
```diff
@@ -61,11 +61,129 @@
...
+            "identifier": "add_departing_employee",
+            "description": "Add a departing employee to the departing employees detection list",
+            "verbose": "There is a short delay before you can query the updated results.",
+            "undo": "remove departing employee",
```
Comment (antazoey) [inline code42.json:69]: I added these `undo` sections - they are neat. They make it so you can right click on old action and there is now an "undo" action available which just guides you through the opposite action

File: code42_connector.py [hunk 3]
```diff
@@ -106,7 +116,63 @@ def _handle_remove_departing_employee(self, param):
...
+        status_message = f"{username} was removed from the high risk employees list"
+        return action_result.set_status(phantom.APP_SUCCESS, status_message)
+
+    def _handle_list_high_risk_employees(self, param):
+        self._log_action_handler()
+        action_result = self._add_action_result(param)
+        filter_type = param.get("filter_type", HighRiskEmployeeFilters.OPEN)
+        results_generator = self.client.detectionlists.high_risk_employee.get_all(filter_type=filter_type)
+        total_count = None
+        for page in results_generator:
```
Comment (alanag13) [inline code42_connector.py:163]: Normally to handle `total_count` I would do something like:

```python
page = None
for page in results_generator:
    employees = page.data.get("items", [])
    for employee in employees: 
         action_result.add_data(employee)
 
total_count = page.data.get("totalCount", 0) if page else None
```
`page` will be set to the last page if there was at least one page of data.

File: code42.json (hunk 2 above)
Comment (timabrmsn) [reply code42.json:69]: that's neat!

File: code42_connector.py (hunk 3 above)
Comment (alanag13) [reply code42_connector.py:163]: you could do this for departing employees too. To me this makes it more clear that we aren't actually setting the total count on each iteration.

**Review by alanag13 — APPROVED** [review]

**Comment by antazoey** [comment]
I have been working with Peter because we have been confused by the summary portion... It seems like the ascii tables are not driven from the summary but rather some little text in the activity list, same place where the `status_message` is... However, when including both a `status_message` and a `summary`, only the `status_message appears` there.  List commands seem to often display the total count and not a `status_message`, so I remove the `status_message` from both list commands.  I have seen other commands from other apps use both the summary and the `status_message`.. The summary data is available in the context...

**Comment by antazoey** [comment]
@alanag13 I made the change discussed via not using status message for list commands and not using summary for add / remove, still approved?

**Pull request merged**